### PR TITLE
Implement GET EventService

### LINKED ISSUE #211: Get EventService
As oneview-redfish-toolkit user, I send a GET request to "/redfish/v1/EventService/" and I receive a JSON containing the main configuration for the EventService.

### PR DESCRIPTION
**Pull request by llvieira**
Closes #211.

### CODE REVIEW

File: oneview_redfish_toolkit/api/event_service.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        self.redfish["@odata.context"] = "/redfish/v1/$metadata#EventService"
+        self.redfish["@odata.id"] = "/redfish/v1/EventService/"
+        self.redfish["Subscriptions"] = dict()
+        self.redfish["Subscriptions"]["@odata.id"] = \
+            "/redfish/v1/EventService/EventSubscriptions/"
+        self.redfish["EventTypesForSubscription"] = \
+            ["StatusChange", "ResourceUpdated", "ResourceAdded",
+             "ResourceRemoved", "Alert"]
+        self.redfish["DeliveryRetryAttempts"] = \
+            int(util.config["event_service"]["DeliveryRetryAttempts"])
```
Comment (ricardoas) [inline oneview_redfish_toolkit/api/event_service.py]: What happens when the user leave this empty in the config file?

File: oneview_redfish_toolkit/api/event_service.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+        self.redfish["DeliveryRetryIntervalSeconds"] = \
+            int(util.config["event_service"]["DeliveryRetryIntervalSeconds"])
+        # All information bellow is hard-coded
+        # until we decide where to find it
+        self.redfish["Status"] = dict()
```
Comment (ricardoas) [inline oneview_redfish_toolkit/api/event_service.py]: Once we have the SCMB connector implemented we can derive this information from that, right?

File: oneview_redfish_toolkit/api/event_service.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        self.redfish["Subscriptions"] = dict()
+        self.redfish["Subscriptions"]["@odata.id"] = \
+            "/redfish/v1/EventService/EventSubscriptions/"
+        self.redfish["EventTypesForSubscription"] = \
+            ["StatusChange", "ResourceUpdated", "ResourceAdded",
+             "ResourceRemoved", "Alert"]
+        self.redfish["DeliveryRetryAttempts"] = \
+            int(util.config["event_service"]["DeliveryRetryAttempts"])
+        self.redfish["DeliveryRetryIntervalSeconds"] = \
+            int(util.config["event_service"]["DeliveryRetryIntervalSeconds"])
```
Comment (ricardoas) [inline oneview_redfish_toolkit/api/event_service.py]: same as above...

File: redfish.conf [hunk 4]
```diff
@@ -15,6 +15,10 @@ api_version = 300
 userName = administrator
 password =
 
+[event_service]
```
Comment (ricardoas) [inline redfish.conf]: Should we add a new section on README????

File: oneview_redfish_toolkit/api/event_service.py [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+        # All information bellow is hard-coded
+        # until we decide where to find it
+        self.redfish["Status"] = dict()
+        self.redfish["Status"]["Health"] = "OK"
+        self.redfish["Status"]["HealthRollup"] = "OK"
+        self.redfish["Status"]["State"] = "Enabled"
+        self.redfish["ServiceEnabled"] = True
```
Comment (ricardoas) [inline oneview_redfish_toolkit/api/event_service.py]: I've been thinking... should we mode this to an utility function (that, for now, will always return True) and then we can only change its implementation?

File: oneview_redfish_toolkit/api/event_service.py (hunk 1 above)
Comment (llvieira) [reply oneview_redfish_toolkit/api/event_service.py]: I moved this code to the blueprint and fixed it.

File: oneview_redfish_toolkit/api/event_service.py (hunk 3 above)
Comment (llvieira) [reply oneview_redfish_toolkit/api/event_service.py]: Fixed!

File: redfish.conf (hunk 4 above)
Comment (llvieira) [reply redfish.conf]: Done!

File: oneview_redfish_toolkit/api/event_service.py (hunk 5 above)
Comment (llvieira) [reply oneview_redfish_toolkit/api/event_service.py]: I created a function called `set_service_status()` and it returns always `True` for now.

File: oneview_redfish_toolkit/api/event_service.py (hunk 2 above)
Comment (llvieira) [reply oneview_redfish_toolkit/api/event_service.py]: I think so...

File: oneview_redfish_toolkit/blueprints/event_service.py [hunk 6]
```diff
@@ -37,11 +39,32 @@ def get_event_service():
...
     """
     try:
+        try:
+            delivery_retry_attempts = \
+                int(util.config["event_service"]["DeliveryRetryAttempts"])
+            delivery_retry_interval = \
+                int(util.
+                    config["event_service"]["DeliveryRetryIntervalSeconds"])
+
+            if delivery_retry_attempts <= 0 or delivery_retry_interval <= 0:
```
Comment (ricardoas) [inline oneview_redfish_toolkit/blueprints/event_service.py]: This seems the kind of thing to check during service launching... wdyt?

File: oneview_redfish_toolkit/blueprints/event_service.py [hunk 7]
```diff
@@ -37,11 +39,32 @@ def get_event_service():
...
+
+        Exceptions:
+            OneViewRedfishError: if there are invalid integers in
+                DeliveryRetryAttempts and DeliveryRetryIntervalSeconds
+                on the CONFIG file.
     """
     try:
+        try:
+            delivery_retry_attempts = \
+                int(util.config["event_service"]["DeliveryRetryAttempts"])
```
Comment (ricardoas) [inline oneview_redfish_toolkit/blueprints/event_service.py]: So we're not using the globals anymore... right?

Comment (llvieira) [reply oneview_redfish_toolkit/blueprints/event_service.py]: I changed that. Now we're using it.

File: oneview_redfish_toolkit/blueprints/event_service.py (hunk 6 above)
Comment (llvieira) [reply oneview_redfish_toolkit/blueprints/event_service.py]: I think so... fixed!

File: oneview_redfish_toolkit/blueprints/event_service.py [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+from oneview_redfish_toolkit.api.errors import OneViewRedfishError
+from oneview_redfish_toolkit.api.event_service \
+    import EventService
+from oneview_redfish_toolkit import util
+
+
+event_service = Blueprint("event_service", __name__)
+
+
+@event_service.route("/redfish/v1/SessionService/", methods=["GET"])
```
Comment (ricardoas) [inline oneview_redfish_toolkit/blueprints/event_service.py]: This is wrong... should be `/redfish/v1/EventService/` already fixed at waffle

**Pull request merged**